apis/debug/data_column_sidecars.yaml

```diff
@@ -0,0 +1,73 @@
+get:
+  operationId: getDebugDataColumnSidecars
+  summary: Get data column sidecars
+  description: |
+    Retrieves data column sidecars for a given block id.
+    Depending on `Accept` header it can be returned either as json or as bytes serialized by SSZ.
+
+    If the `indices` parameter is specified, only the data column sidecars with the specified indices will be returned. There are no guarantees
+    for the returned data column sidecars in terms of ordering.
+  tags:
+    - Debug
+  parameters:
+    - name: block_id
+      in: path
+      required: true
+      $ref: '../../beacon-node-oapi.yaml#/components/parameters/BlockId'
+    - name: indices
+      in: query
+      description: Array of indices for data column sidecars to request for in the specified block. Returns all data column sidecars in the block if not specified.
```

Comment:
Description should clarify that it will return only the columns that this node is custodying. 

Question, if this node has in custody indices [0,1] and the query parameter indices is [1,2]:
- Should the node error, or
- Should the node return [1]